docs/iris/src/userguide/subsetting_a_cube.rst

```diff
@@ -103,6 +103,9 @@ same way as loading with constraints:
 
 Cube iteration
 ^^^^^^^^^^^^^^^
+It is not possible to directly iterate over an Iris cube. That is, you cannot call `cube.next()` as
```

Comment:
I don't think it's very good (modern) Python usage to call a `next` method directly.  Cubes were once iterable by virtue of being indexable #273, but they never did have an actual 'next' function anyway. 
So, I think it would be better here to talk about `next(cube)`, or `list(cube)` or `for x in cube`.

Reply:
@pp-mo Thanks for the review and suggestion. In 2a4f3d2 I've simplified the language slightly, but I think that this should be easily understandable by typical Iris users.